text/0000-cbool.md

```diff
@@ -0,0 +1,28 @@
+- Feature Name: cbool
+- Start Date: Mon Mar  9 00:16:53 CET 2015
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Specify that `bool` is compatible with the `_Bool` C type.
+
+# Motivation
+
+You cannot safely call ffi functions with boolean arguments without a compatible type.
+
+# Detailed design
+
+Specify that `bool` is compatible with the `_Bool` C type and inform the compiler that `bool` is ffi safe.
+
+# Drawbacks
+
+None right now. This should be a no-op on all supported platforms.
```

Comment:
I agree, but @mahkoh needs to do some research (and write the results in the RFC) to ensure that this is as true as possible. (One serious failing of the current RFC is that it doesn't even include the C standard's definition of `_Bool`.)

Also, Rust is an abstract language; it is not just `rustc`. That is, someone may write a Rust compiler that doesn't use LLVM as a backend, and hence not be subject to the restrictions that implies.